consoles/network_console.pm

```diff
@@ -0,0 +1,43 @@
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2019 SUSE LLC
```

Comment:
Looks like a new file. So `# Copyright © 2019 SUSE LLC` should be sufficient.

Reply:
done